src/mlpack/methods/preprocess/preprocess_scale_main.cpp

```diff
@@ -0,0 +1,174 @@
+/**
+ * @file preprocess_scale_main.cpp
+ * @author jeffin sam
+ *
+ * A CLI executable to scale a dataset.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <mlpack/prereqs.hpp>
+#include <mlpack/core/util/mlpack_main.hpp>
+#include <mlpack/core/math/random.hpp>
+#include <mlpack/core/util/cli.hpp>
+#include <mlpack/core/data/scaler_methods/max_abs_scaler.hpp>
+#include <mlpack/core/data/scaler_methods/mean_normalization.hpp>
+#include <mlpack/core/data/scaler_methods/min_max_scaler.hpp>
+#include <mlpack/core/data/scaler_methods/pcawhitening.hpp>
+#include <mlpack/core/data/scaler_methods/zcawhitening.hpp>
+#include <mlpack/core/data/scaler_methods/standard_scaler.hpp>
+#include "mlpack/core/data/scaler_methods/scaling_model.hpp"
+
+using namespace mlpack;
+using namespace mlpack::util;
+using namespace mlpack::data;
+using namespace arma;
+using namespace std;
+
+PROGRAM_INFO("Scale Data",
+    // Short description.
+    "A utility to perform feature scaling.",
```

Comment:
It's ok to use two sentences here---this description is what is seen, e.g., here:
http://mlpack.org/doc/mlpack-3.1.1/cli_documentation.html#gmm_train

(i.e. the two sentences under the call definition)

So you could expand on this here.  Here's an idea...

```
A utility to perform feature scaling on datasets using one of six techniques.  Both scaling and inverse scaling are supported, and scalers can be saved and then applied to other datasets.
```

Maybe you can improve on that a little, it's just an idea. :)

Reply:
You have reviewed few of my PR's and hence you might have noticed, I always have this issue with Documentation, I am not so good at it, Hence i have shamelessly copied your statement :)

Reply:
No worries! :)  Writing documentation is one of the hardest parts in my opinion.  I'm happy to help out where I can.